src/algebra/commute.lean

```diff
@@ -310,9 +320,19 @@ instance centralizer.is_add_submonoid : is_add_submonoid (centralizer a) :=
 { zero_mem := commute.zero_right a,
   add_mem := λ _ _, commute.add_right }
 
+def centralizer.add_submonoid : add_submonoid A :=
```

Comment:
Can this be achieved with `@[to_additive]` on `centralizer.submonoid`? (Also below?)

Reply:
This would necessitate ```@[to_additive]``` on the definitions of ```commute``` and ```semiconj_by``` and a couple of associated lemmas, like ```semiconj_by.eq```, and am I right in thinking ```to_additive```'d versions of ```protected``` lemmas aren't also ```protected``` at the moment? I could just make an explicit ```protected lemma add_semiconj_by.eq``` if necessary though.

Reply:
Wait, my bad. ```@to_additive```ing any amount of lemmas will not make this possible; the underlying set is still the multiplicative centralizer.